etl/pad/raw/skills/en/active_skill_text.py

```diff
@@ -554,6 +554,9 @@ def change_monster(self, act):
         # TODO: Use a template here
         return "Changes to [{}] for the duration of the dungeon".format(act.change_to)
 
+    def random_change_monster(self, act):
+        return "Randomly changes to {} for the duration of the dungeon{}".format(' or '.join(f'[{id}]' for id in act.change_to_ids))
```

Comment:
Do you think you could use `concat_list_and((f'[{id}]' for id in act.change_to_ids), conj='or')`